kobo/apps/trash_bin/models/__init__.py

```diff
@@ -3,6 +3,8 @@
 
```

Comment:
Please remove the extra blank line `from django.utils.timezone import now` should be part of the same block of other django imports.